terraform/alerting/main.tf

```
@@ -7,6 +7,7 @@ module "verification-server" {
   apiserver_hosts = var.apiserver_hosts
   adminapi_hosts  = var.adminapi_hosts
   server_hosts    = var.server_hosts
+	notification-email = var.notification-email
```

```suggestion
	notification-email = var.notification-email
```
  notification-email = var.notification-email

---

I can't tell if that suggestion actually worked, but the whitespace looks off